src/thor/bidirectional_astar.cc

```diff
@@ -1163,11 +1163,13 @@ std::vector<std::vector<PathInfo>> BidirectionalAStar::FormPath(GraphReader& gra
     uint32_t idx1 = edgestatus_forward_.Get(best_connection->edgeid).index();
     uint32_t idx2 = edgestatus_reverse_.Get(best_connection->opp_edgeid).index();
 
+#ifdef LOGGING_LEVEL_TRACE
```

Comment:
we should do `LOGGING_LEVEL_DEBUG` or even get rid of `pathcost` and move the value into the log?